workflow-v2.2.0/tpl/generate_params.toml

```diff
@@ -0,0 +1,175 @@
+#
+# This is the main configuration file for Deis object storage. The values in
+# this file are passed into the appropriate services so that they can configure
+# themselves for persisting data in object storage.
+#
+# In general, all object storage credentials must be able to read and write to
+# the container or bucket they are configured to use.
+#
+# When you change values in this file, make sure to re-run `helmc generate`
+# on this chart.
+
+# Set the storage backend
+#
+# Valid values are:
+# - s3: Store persistent data in AWS S3 (configure in S3 section)
+# - azure: Store persistent data in Azure's object storage
+# - gcs: Store persistent data in Google Cloud Storage
+# - minio: Store persistent data on in-cluster Minio server
+# - swift: Store persistent data in OpenStack Swift storage cluster
+storage = "minio"
+
+# Set the location of Workflow's PostgreSQL database
+#
+# Valid values are:
+# - on-cluster: Run PostgreSQL within the Kubernetes cluster (credentials are generated
+#   automatically; backups are sent to object storage
+#   configured above)
+# - off-cluster: Run PostgreSQL outside the Kubernetes cluster (configure in database section)
+database_location = "on-cluster"
+
+# Set the location of Workflow's logger-specific Redis instance
+#
+# Valid values are:
+# - on-cluster: Run Redis within the Kubernetes cluster
+# - off-cluster: Run Redis outside the Kubernetes cluster (configure in loggerRedis section)
+logger_redis_location = "on-cluster"
+
+[minio]
+org = "deisci"
+pullPolicy = "Always"
+dockerTag = "git-ba66283"
+
+[s3]
+# Your AWS access key. Leave it empty if you want to use IAM credentials.
+accesskey = ""
+# Your AWS secret key. Leave it empty if you want to use IAM credentials.
+secretkey = ""
+# Any S3 region
+region = "us-west-1"
+# Your buckets.
+registry_bucket = "your-registry-bucket-name"
+database_bucket = "your-database-bucket-name"
+builder_bucket = "your-builder-bucket-name"
+
+[azure]
+accountname = "YOUR ACCOUNT NAME"
+accountkey = "YOUR ACCOUNT KEY"
+registry_container = "your-registry-container-name"
+database_container = "your-database-container-name"
+builder_container = "your-builder-container-name"
+
+[gcs]
+# key_json is expanded into a JSON file on the remote server. It must be
+# well-formatted JSON data.
+key_json = '''Paste JSON data here.'''
+registry_bucket = "your-registry-bucket-name"
+database_bucket = "your-database-bucket-name"
+builder_bucket = "your-builder-bucket-name"
+
+[swift]
+username = "Your OpenStack Swift Username"
+password = "Your OpenStack Swift Password"
+authurl = "Swift auth URL for obtaining an auth token"
+# Your OpenStack tenant name if you are using auth version 2 or 3.
+tenant = ""
+authversion = "Your OpenStack swift auth version"
+registry_container = "your-registry-container-name"
+database_container = "your-database-container-name"
+builder_container = "your-builder-container-name"
+
+[builder]
+org = "deisci"
+pullPolicy = "Always"
```

Comment:
>1. copy from the previously released chart
@arschles my reservation with this is there often are changes made in the `workflow-dev` chart that don't get back-ported to any release charts (as I don't believe we back-port any such additions/changes).  Therefore, if we copy from a previous release chart, it may indeed be out-dated.  

I think in this case, I'd vote for `deisrel` to set the pull policy to the appropriate value for any release chart produced from `helm-stage`, if feasible.